force-app/main/default/classes/AFFL_MultiRecordType_TDTM.cls

```diff
@@ -212,31 +318,59 @@ public class AFFL_MultiRecordType_TDTM extends TDTM_Runnable {
             for (String primaryField : afflMapper.primaryAfflFieldNames) {
                 dynamicQuery = dynamicQuery + ', ' + primaryField;
             }
-            dynamicQuery += ' from Contact where ID IN '+ contactIDs;
+            dynamicQuery += ' from Contact where ID IN ' + contactIDs;
             UTIL_Debug.debug('****MRT: Dynamic query: ' + dynamicQuery);
             if (contactIDs != '()') {
                 List<Contact> relatedContactsList = Database.query(dynamicQuery);
-                for(Contact contact : relatedContactsList) {
+                for (Contact contact : relatedContactsList) {
                     relatedContactsMap.put(contact.ID, contact);
                 }
             }
         }
         return relatedContactsMap;
     }
 
-    private void populateKeyAffls(Affiliation__c affl, String lookupFieldAPIName, Contact relatedContact, DmlWrapper dmlWrapper) {
+    /*******************************************************************************************************
```

Comment:
Changes on this method include adding apex doc comments and having the method return a value for testability.